### PR TITLE
tier 30 support for holy priest

### PR DESCRIPTION
**Pull request by foobartles**
### Description

Add support for the Tier 30 4 set for holy priest. I don't think it is possible to track 2 piece because of the way that PoM is logged to the combat log. It would not be clear if a PoM application is tied to 2 piece proc or is just from an existing PoM bouncing.

### Testing
Test report from PTR:
https://www.warcraftlogs.com/reports/AkdM1wpqKVTrLYFG#fight=17
image

### CODE REVIEW

**Comment by trevorm4** [comment]
It's possible to track the 2 piece but it would require a decent amount of coding, you would need to do event linking and apply similar logic to Dancing Mists to track which are from casts and which are from procs (similar idea to dancing mist tracking) since procs will generally apply after the original bounce.

File: src/analysis/retail/priest/holy/modules/dragonflight/tier/tier30/Tier30HolyPriest4Set.tsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    return this.selectedCombatant.getBuffStacks(
+      SPELLS.HOLY_PRIEST_TIER_30_4_SET_BUFF.id,
+      null,
+      FALL_BUFFER,
+    );
+  }
+
+  onSerenityCast(event: CastEvent) {
+    const stacks = this.tierStacks;
+    console.log(stacks);
```
Comment (trevorm4) [inline src/analysis/retail/priest/holy/modules/dragonflight/tier/tier30/Tier30HolyPriest4Set.tsx]: Get rid of this log

File: src/analysis/retail/priest/holy/modules/dragonflight/tier/tier30/Tier30HolyPriest4Set.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      });
+    }
+  }
+
+  onSalvationCast(event: CastEvent) {
+    const stacks = this.tierStacks;
+    if (stacks > 0) {
+      this.stacksOnCast.push(stacks);
+      const healEvents = getSalvationHealEvents(event);
+      console.log(healEvents);
```
Comment (trevorm4) [inline src/analysis/retail/priest/holy/modules/dragonflight/tier/tier30/Tier30HolyPriest4Set.tsx]: Remove

File: src/common/SPELLS/priest.ts [hunk 3]
```diff
@@ -517,6 +517,11 @@ const spells = spellIndexableList({
     name: 'Seize the moment',
     icon: 'inv_mace_1h_artifactheartofkure_d_03',
   },
+  HOLY_PRIEST_TIER_30_4_SET_BUFF: {
+    id: 409479,
+    name: 'Inspired Word',
+    icon: 'ability_paladin_sacredcleansing',
+  },
```
Comment (trevorm4) [inline src/common/SPELLS/priest.ts:524]: Put this in ITEMS/priest.ts instead

**Review by trevorm4 — CHANGES_REQUESTED** [review]

**Comment by foobartles** [comment]
> It's possible to track the 2 piece but it would require a decent amount of coding, you would need to do event linking and apply similar logic to Dancing Mists to track which are from casts and which are from procs (similar idea to dancing mist tracking) since procs will generally apply after the original bounce.

I can look into doing this if I have time, haven't really been playing my priest recently so my motivation for stuff like that has gone down the drain haha

**Comment by trevorm4** [comment]
I saw you requested review but don't see a new commit, did you forget to push?

**Comment by foobartles** [comment]
> I saw you requested review but don't see a new commit, did you forget to push?

🤦 My bad, I hit push but closed my laptop before it finished and the push timed out lol. Sorry about that, should be pushed now

**Review by trevorm4 — APPROVED** [review]

**Pull request merged**